test/suites/integration/webhook_test.go

```diff
@@ -28,170 +28,162 @@ import (
 )
 
 var _ = Describe("Webhooks", func() {
-	Context("Defaulting Webhook", func() {
-		It("should set the default requirements when none are specified", func() {
-			provider := awstest.AWSNodeTemplate(v1alpha1.AWSNodeTemplateSpec{AWS: v1alpha1.AWS{
-				SecurityGroupSelector: map[string]string{"karpenter.sh/discovery": env.ClusterName},
-				SubnetSelector:        map[string]string{"karpenter.sh/discovery": env.ClusterName},
-			}})
-			provisioner := test.Provisioner(test.ProvisionerOptions{
-				ProviderRef: &v1alpha5.ProviderRef{Name: provider.Name},
-			})
-			env.ExpectCreated(provisioner)
-			env.ExpectFound(provisioner)
+	Context("Provisioner", func() {
```

Comment:
I think I'm missing where we actually create the AWSNodeTemplate in some of these tests. Seems like we don't, shouldn't we be?

Reply:
Validation for the provisioner doesn't care about the AWS Node Template. Resources should never validate each other's existence.